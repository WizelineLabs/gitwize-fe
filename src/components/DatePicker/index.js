import React, { useState } from 'react';
import DayPicker, { DateUtils } from 'react-day-picker';
import 'react-day-picker/lib/style.css';
import Paper from '@material-ui/core/Paper'
import FormControl from '@material-ui/core/FormControl'
import { makeStyles } from '@material-ui/core/styles'
import TextField from '@material-ui/core/TextField';
import ClickAwayListener from '@material-ui/core/ClickAwayListener'

const useStyles = makeStyles((theme) => ({
  formControl: {
    marginTop: theme.spacing(2),
    paddingRight: theme.spacing(5),
    width: '100%',
  },
  selectEmpty: {
    marginTop: theme.spacing(2),
  },
  datePicker: {
    position: 'absolute',
    zIndex: 99,
    background: 'rgba(0, 0, 0, 0.85)',
    color: '#ffff',
    marginTop: '10px'
  }
}))

export default function DatePicker(props) {
  const { label, onChange } = props
  const [pickedDate, setPickedDate] = useState({
    from: undefined,
    to: undefined
  })
  const [openDayPickerTable, setOpenDayPickerTable] = useState(false)
  const styles = useStyles()

  const handleClickAway = () => {
    setOpenDayPickerTable(false)
  }

  const toggleDatePicker = () => {
    setOpenDayPickerTable((prev) => !prev);
  }

  const handleDayClick = (day) => {
    const range = DateUtils.addDayToRange(day, pickedDate)
    setPickedDate(range)
    onChange(range)
  }

    const { from, to } = pickedDate
    const modifiers = { start: pickedDate.from, end: pickedDate.to };

    const selectedDateRange = () => {
      if(!from && !to)
        return 'Please select the first day'
      if (from && !to)
        return 'Please select the last day'
      if(from && to)
        return  `Selected from ${from.toLocaleDateString()} - ${to.toLocaleDateString()}`
    }

    return (
      <ClickAwayListener onClickAway={handleClickAway}>
      <FormControl className={styles.formControl}>
        <TextField
          id='standard-helperText'
          label={label}
          value={selectedDateRange()}
          InputProps={{
            readOnly: true,
          }}
          onClick={toggleDatePicker}
        />

      <div className='RangeExample'>
        <Paper style={{display: `${openDayPickerTable ? "block" : "none"}` }} 
          className={styles.datePicker}>
        <DayPicker
          className='Selectable'
          numberOfMonths={2}
          selectedDays={[from, { from, to }]}
          modifiers={modifiers}
          onDayClick={handleDayClick}
          show={false}
        />
        </Paper>
          <style>{`
  .Selectable .DayPicker-Day--selected:not(.DayPicker-Day--start):not(.DayPicker-Day--end):not(.DayPicker-Day--outside) {
    opacity: 0.9;
    background: linear-gradient(0deg, rgba(0, 0, 0, 0.9), rgba(0, 0, 0, 0.9)), #000000;
  }
  .Selectable .DayPicker-Day {
    border-radius: 0 !important;
  }
  .Selectable .DayPicker-Day--start {
    border-radius: 100% !important;
  }
  .Selectable .DayPicker-Day--end {
    border-radius: 100% !important;
  }
  .Selectable .DayPicker-Month {
    width: 100%
  }

  .Selectable .DayPicker-Day--selected:not(.DayPicker-Day--disabled):not(.DayPicker-Day--outside) {
    background: #EC5D5C;
  }

`}</style>
      </div>
      </FormControl>
      </ClickAwayListener>
    );
}