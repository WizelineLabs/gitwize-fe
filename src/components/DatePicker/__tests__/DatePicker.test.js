import React from 'react'
import {mount, shallow} from 'enzyme'
import DatePicker from '../index'
import renderer from 'react-test-renderer'

describe('DatePicker', () => {
  const labelName = 'Date Range'

  test('render the component correctly', () => {
    const onChange = jest.fn(),
      props = {
        label: labelName,
        onChange
      }

    const wrapper = mount(<DatePicker {...props} />);
    expect(wrapper.find(DatePicker).exists()).toEqual(true)
    expect(wrapper.find(DatePicker).prop('label')).toEqual(labelName)
  })

  test('check dropdown displayed', () => {
    const onChange = jest.fn(),
      props = {
        label: labelName,
        onChange
      }


    const wrapper = renderer.create(<DatePicker {...props} />).toJSON()

    expect(wrapper).toMatchSnapshot()
    //const textField = wrapper.find('#standard-helperText')
    //expect(textField).exists().toEqual(true)

  })

  test('check the onChange callback', () => {

    const onChange = jest.fn(),
      props = {
        label: "Date Range",
        onChange
      }

    const DatePickerComponent = mount(<DatePicker {...props} />)
  })


})