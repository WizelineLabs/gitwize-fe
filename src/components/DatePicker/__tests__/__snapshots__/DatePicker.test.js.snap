// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatePicker check dropdown displayed 1`] = `
<div
  className="MuiFormControl-root makeStyles-formControl-1"
  onClick={[Function]}
  onTouchEnd={[Function]}
>
  <div
    className="MuiFormControl-root MuiTextField-root"
    onClick={[Function]}
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiFormLabel-filled"
      data-shrink={true}
      htmlFor="standard-helperText"
      id="standard-helperText-label"
    >
      Date Range
    </label>
    <div
      className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        autoFocus={false}
        className="MuiInputBase-input MuiInput-input"
        disabled={false}
        id="standard-helperText"
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        readOnly={true}
        required={false}
        type="text"
        value="Please select the first day"
      />
    </div>
  </div>
  <div
    className="RangeExample"
  >
    <div
      className="MuiPaper-root makeStyles-datePicker-3 MuiPaper-elevation1 MuiPaper-rounded"
      style={
        Object {
          "display": "none",
        }
      }
    >
      <div
        className="DayPicker Selectable"
        lang="en"
      >
        <div
          className="DayPicker-wrapper"
          onKeyDown={[Function]}
          tabIndex={0}
        >
          <div
            className="DayPicker-NavBar"
          >
            <span
              aria-label="Previous Month"
              className="DayPicker-NavButton DayPicker-NavButton--prev"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              tabIndex="0"
            />
            <span
              aria-label="Next Month"
              className="DayPicker-NavButton DayPicker-NavButton--next"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
              tabIndex="0"
            />
          </div>
          <div
            className="DayPicker-Months"
          >
            <div
              className="DayPicker-Month"
              role="grid"
            >
              <div
                className="DayPicker-Caption"
                role="heading"
              >
                <div
                  onKeyUp={[Function]}
                >
                  July 2020
                </div>
              </div>
              <div
                className="DayPicker-Weekdays"
                role="rowgroup"
              >
                <div
                  className="DayPicker-WeekdaysRow"
                  role="row"
                >
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Sunday"
                    >
                      Su
                    </abbr>
                  </div>
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Monday"
                    >
                      Mo
                    </abbr>
                  </div>
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Tuesday"
                    >
                      Tu
                    </abbr>
                  </div>
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Wednesday"
                    >
                      We
                    </abbr>
                  </div>
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Thursday"
                    >
                      Th
                    </abbr>
                  </div>
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Friday"
                    >
                      Fr
                    </abbr>
                  </div>
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Saturday"
                    >
                      Sa
                    </abbr>
                  </div>
                </div>
              </div>
              <div
                className="DayPicker-Body"
                role="rowgroup"
              >
                <div
                  className="DayPicker-Week"
                  role="row"
                >
                  <div
                    aria-disabled={true}
                    className="DayPicker-Day DayPicker-Day--outside"
                  />
                  <div
                    aria-disabled={true}
                    className="DayPicker-Day DayPicker-Day--outside"
                  />
                  <div
                    aria-disabled={true}
                    className="DayPicker-Day DayPicker-Day--outside"
                  />
                  <div
                    aria-disabled={false}
                    aria-label="Wed Jul 01 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={0}
                  >
                    1
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Thu Jul 02 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    2
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Fri Jul 03 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    3
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Sat Jul 04 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    4
                  </div>
                </div>
                <div
                  className="DayPicker-Week"
                  role="row"
                >
                  <div
                    aria-disabled={false}
                    aria-label="Sun Jul 05 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    5
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Mon Jul 06 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    6
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Tue Jul 07 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    7
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Wed Jul 08 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    8
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Thu Jul 09 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    9
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Fri Jul 10 2020"
                    aria-selected={false}
                    className="DayPicker-Day DayPicker-Day--today"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    10
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Sat Jul 11 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    11
                  </div>
                </div>
                <div
                  className="DayPicker-Week"
                  role="row"
                >
                  <div
                    aria-disabled={false}
                    aria-label="Sun Jul 12 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    12
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Mon Jul 13 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    13
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Tue Jul 14 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    14
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Wed Jul 15 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    15
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Thu Jul 16 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    16
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Fri Jul 17 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    17
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Sat Jul 18 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    18
                  </div>
                </div>
                <div
                  className="DayPicker-Week"
                  role="row"
                >
                  <div
                    aria-disabled={false}
                    aria-label="Sun Jul 19 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    19
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Mon Jul 20 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    20
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Tue Jul 21 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    21
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Wed Jul 22 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    22
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Thu Jul 23 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    23
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Fri Jul 24 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    24
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Sat Jul 25 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    25
                  </div>
                </div>
                <div
                  className="DayPicker-Week"
                  role="row"
                >
                  <div
                    aria-disabled={false}
                    aria-label="Sun Jul 26 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    26
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Mon Jul 27 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    27
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Tue Jul 28 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    28
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Wed Jul 29 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    29
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Thu Jul 30 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    30
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Fri Jul 31 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    31
                  </div>
                  <div
                    aria-disabled={true}
                    className="DayPicker-Day DayPicker-Day--outside"
                  />
                </div>
              </div>
            </div>
            <div
              className="DayPicker-Month"
              role="grid"
            >
              <div
                className="DayPicker-Caption"
                role="heading"
              >
                <div
                  onKeyUp={[Function]}
                >
                  August 2020
                </div>
              </div>
              <div
                className="DayPicker-Weekdays"
                role="rowgroup"
              >
                <div
                  className="DayPicker-WeekdaysRow"
                  role="row"
                >
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Sunday"
                    >
                      Su
                    </abbr>
                  </div>
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Monday"
                    >
                      Mo
                    </abbr>
                  </div>
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Tuesday"
                    >
                      Tu
                    </abbr>
                  </div>
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Wednesday"
                    >
                      We
                    </abbr>
                  </div>
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Thursday"
                    >
                      Th
                    </abbr>
                  </div>
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Friday"
                    >
                      Fr
                    </abbr>
                  </div>
                  <div
                    className="DayPicker-Weekday"
                    role="columnheader"
                  >
                    <abbr
                      title="Saturday"
                    >
                      Sa
                    </abbr>
                  </div>
                </div>
              </div>
              <div
                className="DayPicker-Body"
                role="rowgroup"
              >
                <div
                  className="DayPicker-Week"
                  role="row"
                >
                  <div
                    aria-disabled={true}
                    className="DayPicker-Day DayPicker-Day--outside"
                  />
                  <div
                    aria-disabled={true}
                    className="DayPicker-Day DayPicker-Day--outside"
                  />
                  <div
                    aria-disabled={true}
                    className="DayPicker-Day DayPicker-Day--outside"
                  />
                  <div
                    aria-disabled={true}
                    className="DayPicker-Day DayPicker-Day--outside"
                  />
                  <div
                    aria-disabled={true}
                    className="DayPicker-Day DayPicker-Day--outside"
                  />
                  <div
                    aria-disabled={true}
                    className="DayPicker-Day DayPicker-Day--outside"
                  />
                  <div
                    aria-disabled={false}
                    aria-label="Sat Aug 01 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={0}
                  >
                    1
                  </div>
                </div>
                <div
                  className="DayPicker-Week"
                  role="row"
                >
                  <div
                    aria-disabled={false}
                    aria-label="Sun Aug 02 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    2
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Mon Aug 03 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    3
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Tue Aug 04 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    4
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Wed Aug 05 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    5
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Thu Aug 06 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    6
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Fri Aug 07 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    7
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Sat Aug 08 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    8
                  </div>
                </div>
                <div
                  className="DayPicker-Week"
                  role="row"
                >
                  <div
                    aria-disabled={false}
                    aria-label="Sun Aug 09 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    9
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Mon Aug 10 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    10
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Tue Aug 11 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    11
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Wed Aug 12 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    12
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Thu Aug 13 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    13
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Fri Aug 14 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    14
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Sat Aug 15 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    15
                  </div>
                </div>
                <div
                  className="DayPicker-Week"
                  role="row"
                >
                  <div
                    aria-disabled={false}
                    aria-label="Sun Aug 16 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    16
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Mon Aug 17 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    17
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Tue Aug 18 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    18
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Wed Aug 19 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    19
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Thu Aug 20 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    20
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Fri Aug 21 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    21
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Sat Aug 22 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    22
                  </div>
                </div>
                <div
                  className="DayPicker-Week"
                  role="row"
                >
                  <div
                    aria-disabled={false}
                    aria-label="Sun Aug 23 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    23
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Mon Aug 24 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    24
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Tue Aug 25 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    25
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Wed Aug 26 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    26
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Thu Aug 27 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    27
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Fri Aug 28 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    28
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Sat Aug 29 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    29
                  </div>
                </div>
                <div
                  className="DayPicker-Week"
                  role="row"
                >
                  <div
                    aria-disabled={false}
                    aria-label="Sun Aug 30 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    30
                  </div>
                  <div
                    aria-disabled={false}
                    aria-label="Mon Aug 31 2020"
                    aria-selected={false}
                    className="DayPicker-Day"
                    onClick={[Function]}
                    onKeyDown={[Function]}
                    role="gridcell"
                    tabIndex={-1}
                  >
                    31
                  </div>
                  <div
                    aria-disabled={true}
                    className="DayPicker-Day DayPicker-Day--outside"
                  />
                  <div
                    aria-disabled={true}
                    className="DayPicker-Day DayPicker-Day--outside"
                  />
                  <div
                    aria-disabled={true}
                    className="DayPicker-Day DayPicker-Day--outside"
                  />
                  <div
                    aria-disabled={true}
                    className="DayPicker-Day DayPicker-Day--outside"
                  />
                  <div
                    aria-disabled={true}
                    className="DayPicker-Day DayPicker-Day--outside"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <style>
      
  .Selectable .DayPicker-Day--selected:not(.DayPicker-Day--start):not(.DayPicker-Day--end):not(.DayPicker-Day--outside) {
    opacity: 0.9;
    background: linear-gradient(0deg, rgba(0, 0, 0, 0.9), rgba(0, 0, 0, 0.9)), #000000;
  }
  .Selectable .DayPicker-Day {
    border-radius: 0 !important;
  }
  .Selectable .DayPicker-Day--start {
    border-radius: 100% !important;
  }
  .Selectable .DayPicker-Day--end {
    border-radius: 100% !important;
  }
  .Selectable .DayPicker-Month {
    width: 100%
  }

  .Selectable .DayPicker-Day--selected:not(.DayPicker-Day--disabled):not(.DayPicker-Day--outside) {
    background: #EC5D5C;
  }


    </style>
  </div>
</div>
`;
